spec/input_validator_spec.rb

```diff
@@ -22,71 +22,46 @@
         end      
     end
 
-    describe "#get_invalid_moves" do
-        it "should list all positions that have previously been requested" do
-            board = Board.new
-
-            board.move(2)
-            board.move(3)
-            invalid_moves = [2,3]
-
-            expect(@input_v.get_invalid_moves(board)).to eq(invalid_moves)
-        end
-
-        it "should list all positions that have previously been requested" do
-            board = Board.new
-
-            board.move(2)
-            board.move(3)
-            board.move(1)
-            board.move(4)
-            invalid_moves = [2,3,1,4]
-
-            expect(@input_v.get_invalid_moves(board)).to eq(invalid_moves)
-        end
-    end
-
     describe "#valid_move?" do
-        it "should return false if requested move is already in the board's moves array" do
-            board = Board.new
-
-            board.move(2)
-            @input_v.get_invalid_moves(board)
-            expect(@input_v.valid_move?(2)).to eq(false)
+        before(:each) do
+            @board = Board.new
         end
 
-        it "should return false if requested move is already in the board's moves array" do
-            board = Board.new
+        it "should return false if one requested move is already in the board's moves array" do
+            token = "X"
+            @board.move(token, 2)
+            
+            expect(@input_v.valid_move?(@board.moves, 2)).to eq(false)
+        end
 
-            board.move(2)
-            board.move(3)
-            board.move(1)
-            board.move(4)
-            @input_v.get_invalid_moves(board)
-            expect(@input_v.valid_move?(2)).to eq(false)
-            expect(@input_v.valid_move?(1)).to eq(false)
-            expect(@input_v.valid_move?(1)).to eq(false)
-            expect(@input_v.valid_move?(4)).to eq(false)
+        it "should return false if many requested moves are already in the board's moves array" do
```

Comment:
This description doesn't make a lot of sense to me 🤔

Reply:
I deleted this test.